docs/src/user-manual/ingest/index.md

```diff
@@ -0,0 +1,38 @@
+# Ingest
+
+This section of the User manual covers the [Ingest](../glossary.md#ingest)
+domain functionality.
+
+In Enduro, **ingest** is defined as a phase in a preservation workflow
+describing all the preservation policy-defined tasks performed on a SIP when it
+is received from a producer, prior to preservation. Typically this phase covers
+**validation** activities (performed against SIP files, structure, and/or
+metadata) as well as any **package transformations** (removal of unneeded or
+temporary files, restructuring, etc) to optimize the package for further
+processing by the preservation engine.
+
+At installation, Enduro's default ingest functionality is minimal - the
+application can receive and unpack SIPs, perform basic file format validation
+for some file types, and then restructure and deliver the package for
+preservation with either [Archivematica][Archivematica] or [a3m][a3m]. However,
+Enduro's workflows are intended to be customized via the addition of **child
+workflow activities**, which can be designed to implement the specific ingest
+needs of a given organization.
+
+For further information on child workflows and how to develop and implement
+custom activities into your Enduro workflows, see:
+
+* [Preprocessing child workflow](../../dev-manual/preprocessing.md)
```

Comment:
Instead of separating this out into its own bullet, I'd add the hyperlink to "child workflow activities" above on lines 18-19

Reply:
Done